src/timepicker/docs/readme.md

```diff
@@ -0,0 +1,11 @@
+In the time entry field allow user to enter the time in several formats. When user have entered the value, change formatting to the 24:00 clock. 
+### The format needed to be supported are:  ###
+ * 8a = 08:00 (If last character is a, the time is am) 
+ * 8p = 20:00 (If last character is p the time is pm)
+ * 815a / p = 08:15 / 20:15
+ * 8.15a = 08:15 / 20:15
+ * 8 = 08:00
+ * 815 = 08:15
```

Comment:
yeah, it might be good because when you have "815" you don't have anything called "81" hours and "5" minutes so that's rather clear. But it's not clear if "125" would be "1" hour and "25" minutes or "12" hours and "50" minutes